Examples/Examples-tvOS/IGListKitExamples/Views/LabelCell.swift

```diff
@@ -25,7 +25,7 @@ final class LabelCell: UICollectionViewCell {
 
     static func textHeight(_ text: String, width: CGFloat) -> CGFloat {
         let constrainedSize = CGSize(width: width - insets.left - insets.right, height: CGFloat.greatestFiniteMagnitude)
-        let attributes = [ NSFontAttributeName: font ]
+        let attributes = [ NSAttributedStringKey.font: font ]
```

Comment:
Should be able to drop enum type here and let it infer it `.font:`

Reply:
Nice, ya this was the migration tool